WcaOnRails/app/webpacker/components/Panel/Board/RegionManager.jsx

```diff
@@ -45,50 +49,130 @@ function UserGroupVisibility({ userGroup, save, sync }) {
 
 export default function RegionManager() {
   const {
-    data, loading, error, sync,
+    data, loading: fetchLoading, error, sync,
   } = useLoadedData(fetchUserGroupsUrl('delegate_regions'));
   const { save, saving } = useSaveAction();
   const [openModalType, setOpenModalType] = React.useState();
   const [newRegion, setNewRegion] = React.useState(defaultRegion);
   const [saveError, setSaveError] = React.useState();
+  const [selectedGroup, setSelectedGroup] = React.useState();
+  const [newLeadDelegate, setNewLeadDelegate] = React.useState();
+  const [loading, setLoading] = React.useState(false);
+
+  React.useEffect(() => {
+    if (selectedGroup) {
+      setOpenModalType('newLeadDelegate');
+    }
+  }, [selectedGroup]);
 
-  const closeModal = () => setOpenModalType(null);
+  const closeModal = () => {
+    setOpenModalType(null);
+  };
```

Comment:
Any reason this was changed?

Reply:
Oops did it accidentally. Thanks for pointing.